rust/src/lib.rs

```diff
@@ -15,6 +15,7 @@
  * 02110-1301, USA.
  */
 
+#![allow(ellipsis_inclusive_range_patterns)]
```

Comment:
Please add a comment with a link to the ticket.

We should add a reference to this in https://redmine.openinfosecfoundation.org/issues/2629 so that we can removed it when we switch to Rust 2018 edition.